pkg/csi/service/common/vsphereutil.go

```diff
@@ -520,6 +596,44 @@ func CreateFileVolumeUtilOld(ctx context.Context, clusterFlavor cnstypes.CnsClus
 		}
 	}
 
+	if spec.StoragePolicyID != "" {
```

Comment:
why do we need to maintain `CreateFileVolumeUtilOld`?
Can't we remove this function?

Reply:
same here. I saw a lot of duplicated code in this change.

Reply:
Removed unused CreateFileVolumeUtilOld function, and removed duplicated code